src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl

```diff
@@ -28,6 +28,77 @@
     "description": "Design review"
   },
   "paths": {
+    "/payment-methods/{id}/sessions": {
```

Comment:
why add this v2 api version? an optional header have been added to api so no breaking changes.
optional header can be add to the v1 version.

what do you think?

Reply:
apply the new header only to v2 (and eventually move from v1 to v2 the required APIs) was decided during the refinement of the task.